app/jobs/rate_limit_checker_job.rb

```diff
@@ -1,5 +1,6 @@
-class RateLimitCheckerJob < ApplicationJob
-  queue_as :rate_limit_checker
+class RateLimitCheckerJob
```

Comment:
I was wondering if it wouldn't be better to separate the PR that adds Sidekiq from the PR that starts switching jobs. 

Another thing: don't we have to make sure there are no `RateLimitCheckerJob` running before we switch this over? What happens if after the restart of the dynos`ActiveJob/Delayed::Job` loads the `RateLimitCheckerJob` expecting it to be an `ActiveJob` class?

Reply:
If there were jobs on the queue they would fail and we would have to manually run them ourselves. In the case of this job I am not concerned about that happening. In the case of indexing jobs we will want to keep the original job, switch all code calls to the new job then let the DJ queues drain before we removed the old job. Many jobs that don't run frequently however or are not super critical I think switching full over in one swoop is low risk.

In order to make new Sidekiq jobs while perserving the DJ ones we will have to name them differently. **What do you all think about making all jobs switched to Sidekiq end in `worker` instead of job and that indicates they have been switched?** In other words this would become `RateLimitCheckerWorker`

Reply:
This idea seems to fit into sidekiq naming conventions, so I'm good with it.

I agree that moving jobs to sidekiq should be in separate prs.

Reply:
I would adapt the same strategy for all for consistency even if it might be superfluous for some jobs. It creates examples in case extern contributors want to help and helps all of us that have less visibility understand what's going on plus the obvious safety of letting the queue drain by itself instead of having to rerun jobs manually because we don't have the old job anymore